### ISSUE TITLE
GPL-939 sentinel prevents controls being picked into well D3 or well H10 of the 96-well cherrypick plate [Heron]

### ISSUE THREAD

**Issue by rl15** [sanger/sequencescape#3134]
**User story**
As a member of R&D I would like to add code into the sentinel pipeline which prevents heron controls being picked into well D3 or well H10 of the 96-well cherrypick plate.

**Who are the primary contacts for this story**
Lesley S

**Acceptance criteria**
To be considered successful the solution must allow:
- [x] Placing of controls continues to be randomised across the remainder of the plates.

**Additional context**

Changed required by 6th April if achievable (2 weeks from mail of 23rd March)

**Comment by JamesGlover** [comment]
Some residual code in the Sentinal pipeline to keep wells clear, although think it was mostly geared up to protect particular columns, so may mostly be concerned with blocking out the initial x wells.

**Comment by JamesGlover** [comment]
Added a configuration option: 

`Rails.application.config.plate_default_control_wells_to_leave_free`

That lets us quickly switch the wells to leave free. Currently has a default matching these requirements, so don't need to update deployment project.

I decided against getting more clever, as while I'm certain we'll need to change this at some point again, it's not 100% clear what form the changes will take. Meanwhile, pushing it up to yet another option in cherrypicking just increases complexity for the user, and increases the risk that they will make a mistake.

My suspicion is that at some point we will need to very which wells get left free based on the process downstream, which would ideally be driven by the target purpose.
